tldw_Server_API/app/core/TTS/tts_service_v2.py

```
@@ -246,6 +258,136 @@ async def list_providers(self) -> List[str]:
+    async def get_capabilities(self) -> Dict[str, Any]:
+        """
+        Return capabilities for all available TTS providers.
+
+        The structure is JSON-serializable and suitable for the
+        `/api/v1/audio/providers` endpoint.
+        """
+        capabilities: Dict[str, Any] = {}
+
+        try:
+            factory = await self._ensure_factory()
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)